README.md

```diff
@@ -2,8 +2,10 @@
 ---------------------------
 Source files in order to conduct an introduction to the command line workshop
 
+Source Material for Mentorship Programme.  (Student : Daniel M)
```

Comment:
@DanielNMartin students are not allowed to write there name in these classified documents, please remove name and re-submit pull request